wolfcrypt/benchmark/benchmark.c

```diff
@@ -1011,15 +1040,25 @@ static const char* bench_Usage_msg1[][22] = {
         "<num>       ブロックサイズをバイト単位で指定します。\n",
         "-blocks <num>  TBD.\n",
         "-threads <num> 実行するスレッド数\n",
-        "-print      ベンチマーク統計の要約を表示する\n"
+        "-print      ベンチマーク統計の要約を表示する\n",
+        /* TODO: translate below */
+        "-hash_input   <file>   Input data to use for hash benchmarking\n",
+        "-cipher_input <file>   Input data to use for cipher benchmarking\n"
```

Comment:
Missing comma at the end of the line.  And nope, I didn't spot that with my bare eyes -- thank `clang-tidy` `bugprone-suspicious-missing-comma` for that catch.